packages/web-twig/src/Resources/components/FileUploader/FileUploaderInput.twig

```diff
@@ -4,6 +4,7 @@
 {%- set _helperText = props.helperText | default(null) -%}
 {%- set _iconName = props.iconName | default('upload') -%}
 {%- set _id = props.id -%}
+{%- set _isDisabled = props.isDisabled | default(false) | boolprop -%}
```

Comment:
⚠️ We do not want to use the `boolprop` anymore IMHO. The user should use the correct value otherwise it should blow up. I suggest removing all three `boolprop` filters.